package.json

```diff
@@ -5,19 +5,17 @@
   "main": "./build/index.js",
   "dependencies": {
     "classnames": "^2.2.5",
-    "css-loader": "^0.26.1",
-    "expect": "^1.20.2",
-    "file-loader": "^0.10.0",
-    "lodash": "4.17.4",
+    "css-loader": "^0.26.4",
+    "file-loader": "^0.10.1",
+    "lodash": "^4.17.4",
     "prop-types": "^15.6.0",
-    "react": "^16.1.0",
-    "react-dom": "^16.1.0",
-    "react-onclickoutside": "^6.6.2",
+    "react": "^16.2.0",
```

Comment:
It is a bad practice to update many dependencies, especially if they are so important. Other developers can have lots of problems with resolving those, especially by using them with other plug-ins.

Reply:
Sorry, the file got checked in automatically by mistake. I only needed to change the promise method so it's jus tone source file change.

Reply:
Ok, no problem - we just trying to go through normal review process.